src/modules/dashboard/components/Vesting/UnwrapTokensPage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { defineMessage, FormattedMessage } from 'react-intl';
+
+import Heading from '~core/Heading';
+
+import VestingPageLayout from './VestingPageLayout';
+
+const MSG = defineMessage({
+  title: {
+    id: 'dashboard.Vesting.UnwrapTokensPage.title',
+    defaultMessage: 'Unwrap w{tokenSymbol}',
+  },
+  wrappedBalance: {
+    id: 'dashboard.Vesting.UnwrapTokensPage.wrappedBalance',
+    defaultMessage: 'w{tokenSymbol} balance',
+  },
+  balance: {
+    id: 'dashboard.Vesting.UnwrapTokensPage.balance',
+    defaultMessage: '{tokenSymbol} balance',
+  },
+  buttonUnwrap: {
+    id: 'dashboard.Vesting.UnwrapTokensPage.buttonUnwrap',
+    defaultMessage: 'Unwrap {tokenSymbol}',
+  },
+});
+
+const displayName = 'dashboard.Vesting.UnwrapTokensPage';
+
+const UnwrapTokensPage = () => {
+  // replace with a query later
+  const token = {
+    symbol: 'CLNY',
+    wrappedBalance: '70000000000000000000000',
+    balance: '300000000000000000000',
+    decimals: 18,
+  };
+
+  return (
+    <VestingPageLayout
+      // Add proper loading state when connected to queries
+      isLoading={false}
+      title={
+        <Heading
+          appearance={{ size: 'medium', theme: 'dark' }}
+          text={MSG.title}
+          textValues={{ tokenSymbol: token?.symbol }}
+        />
+      }
+      tableValues={[
+        {
+          label: (
+            <FormattedMessage
+              {...MSG.wrappedBalance}
+              values={{ tokenSymbol: token?.symbol }}
+            />
+          ),
+          value: token.wrappedBalance,
```

Comment:
Should there be some checks and fallbacks, in case no token data is retrieved from query?

Reply:
Or, will that be handled by the wiring?

Reply:
Handled by wiring, but even then I don't think I'll put in a fallback for this.

If the token doesn't load, there's bigger problems as the colony, prior to this hasn't loaded so the dapp crashes :)